cni/network/network.go

```diff
@@ -801,13 +875,15 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 
 		epInfos = append(epInfos,
 			&network.EndpointInfo{
+				IfName:            epInfo.IfName,
 				ContainerID:       epInfo.ContainerID,
 				NetNsPath:         epInfo.NetNsPath,
 				IPAddresses:       addresses,
 				Routes:            secondaryCniResult.Routes,
 				MacAddress:        secondaryCniResult.MacAddress,
 				NICType:           secondaryCniResult.NICType,
 				SkipDefaultRoutes: secondaryCniResult.SkipDefaultRoutes,
+				Policies:          opt.policies,
```

Comment:
can you check if we are setting this

Reply:
we get it from conflist file with outbound NAT policies that I confirmed